ssl_website_check.sh

```diff
@@ -13,19 +13,30 @@ DOMAINS=(
 'gmail-smtp-in.l.google.com. 25 smtp'
 )
 
+DOMAINS_LIST="google.de yahoo.com flickr.com"
+#DOMAIN_LIST=$(ls /home/user100domain/data/www | egrep -v 'tar|gz|bz2|::')
+
+ping_packet_count="2"
+
 function check_ssl_cert()
 {
     host=$1
-    port=$2
+    port=${2:-"443"}
     proto=$3
 
+	[ -z $host ] && break;
     if [ -n "$proto" ]
     then
         starttls="-starttls $proto"
     else
         starttls=""
     fi
 
+	if (! ping -q -c${ping_packet_count} $host > /dev/null 2>&1); then
+	    printf "| %30s | %5s | %-109s |\n" "$host" "$port" "No ping to host!"
+		continue;
```

Comment:
Since we are in a function here, `continue` doesn't make any sense here. Use `return`.